package.json

```diff
@@ -56,6 +56,7 @@
     "semver": "5.5.0",
     "socket.io": "2.0.4",
     "spdy": "3.4.7",
+    "sqlite3": "3.1.13",
```

Comment:
For good measure, I went through a couple of their (*many*) open issues, and oh boy does it feel like `node-ldapjs` again (if I'm reading the following tickets correctly):

- [Their README](https://github.com/mapbox/node-sqlite3#supported-platforms) doesn't mention Node v8 or Node v9 support, though their CI seems fine with it (also see https://github.com/mapbox/node-sqlite3/issues/939 and https://github.com/mapbox/node-sqlite3/issues/906)
- https://github.com/mapbox/node-sqlite3/issues/902 mentions issues with Node v9
- https://github.com/mapbox/node-sqlite3/issues/903 and https://github.com/mapbox/node-sqlite3/issues/942 talk about issues with the RPi3
- https://github.com/mapbox/node-sqlite3/issues/910 mentions issues with Win32
- The eternal `node-gyp` issues with https://github.com/mapbox/node-sqlite3/issues/909, https://github.com/mapbox/node-sqlite3/issues/901 (I stopped counting projects where people reported errors due to `node-gyp` thingies)

Ouch.
Note that I'm not trying to make some sort of a point with this comment, just documenting findings. Just like when we introduced LDAP support that got us a few install-related tickets, we might have to face issues related to installs, especially if this is not compatible with v8/v9!

Just putting this out there, someone mentioned https://github.com/JoshuaWise/better-sqlite3 as an alternative 🤷‍♂️

Given all those, I think we really want some minimal tests around writing/loading logs, so that all supported Node versions and Linux/Windows support get some confidence that this is working.
Thoughts?